stable/xray/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # JFrog Xray Chart Changelog
 All changes to this chart will be documented in this file
 
+## [6.4.2] - Nov 16, 2020
+* Update Xray to version 3.11.2 - [Release Notes](https://www.jfrog.com/confluence/display/JFROG/Xray+Release+Notes#XrayReleaseNotes-Xray3.11)
```

Comment:
Link for Xray 3.11.2 relaese note is : https://www.jfrog.com/confluence/display/JFROG/Xray+Release+Notes#XrayReleaseNotes-Xray3.11.2

Reply:
yes, Since we don't have 3.11.0 , 3.11.1 in bintray , hence defaulted to 3.11 release notes